packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+
+%dir %{_cross_libdir}/systemd
+%{_cross_libdir}/systemd/libsystemd-core-*.so
+%{_cross_libdir}/systemd/libsystemd-shared-*.so
+%{_cross_libdir}/systemd/systemd
+%{_cross_libdir}/systemd/systemd-boot-check-no-failures
+%{_cross_libdir}/systemd/systemd-cgroups-agent
+%{_cross_libdir}/systemd/systemd-executor
+%{_cross_libdir}/systemd/systemd-fsck
+%{_cross_libdir}/systemd/systemd-growfs
+%{_cross_libdir}/systemd/systemd-journald
+%{_cross_libdir}/systemd/systemd-logind
+%{_cross_libdir}/systemd/systemd-makefs
+%{_cross_libdir}/systemd/systemd-modules-load
+%{_cross_libdir}/systemd/systemd-network-generator
+%{_cross_libdir}/systemd/systemd-pstore
+%{_cross_libdir}/systemd/systemd-random-seed
+%{_cross_libdir}/systemd/systemd-remount-fs
+%{_cross_libdir}/systemd/systemd-shutdown
+%{_cross_libdir}/systemd/systemd-sleep
+%{_cross_libdir}/systemd/systemd-socket-proxyd
+%{_cross_libdir}/systemd/systemd-sysctl
+%{_cross_libdir}/systemd/systemd-sysroot-fstab-check
+%{_cross_libdir}/systemd/systemd-udevd
+%{_cross_libdir}/systemd/profile.d/70-systemd-shell-extra.sh
+
+%exclude %{_cross_libdir}/systemd/repart/*
+%exclude %{_cross_libdir}/systemd/system-generators/systemd-ssh-generator
+%exclude %{_cross_libdir}/systemd/system-generators/systemd-tpm2-generator
+
+%exclude %{_cross_libdir}/systemd/system/tpm2.target
+%exclude %{_cross_libdir}/systemd/systemd-battery-check
+%exclude %{_cross_libdir}/systemd/systemd-ssh-proxy
+
+%exclude %{_cross_libdir}/systemd/system/capsule.slice
+%exclude %{_cross_libdir}/systemd/system/capsule@.service
+%exclude %{_cross_libdir}/systemd/system/sockets.target.wants/systemd-creds.socket
+%exclude %{_cross_libdir}/systemd/system/soft-reboot.target
+%exclude %{_cross_libdir}/systemd/system/ssh-access.target
+%exclude %{_cross_libdir}/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup-dev-early.service
+%exclude %{_cross_libdir}/systemd/system/systemd-creds.socket
+%exclude %{_cross_libdir}/systemd/system/systemd-creds@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-growfs-root.service
+%exclude %{_cross_libdir}/systemd/system/systemd-growfs@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-journald-sync@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-soft-reboot.service
+%exclude %{_cross_libdir}/systemd/system/systemd-tmpfiles-setup-dev-early.service
+%exclude %{_cross_libdir}/systemd/system/systemd-udev-load-credentials.service
+
+# Disable the notifier service: systemd-networkd-persistent-storage.service.
+# Without this service Bottlerocket loses support for reusing DHCP leases on
+# reboot but avoids setting up a storage dependency for systemd-networkd
+%exclude %{_cross_libdir}/systemd/system/systemd-networkd-persistent-storage.service
+
+
+%exclude %{_cross_libdir}/udev/iocost
+%exclude %{_cross_libdir}/udev/rules.d/60-dmi-id.rules
+%exclude %{_cross_libdir}/udev/rules.d/60-persistent-storage-mtd.rules
+%exclude %{_cross_libdir}/udev/rules.d/90-iocost.rules
```

Comment:
Let's keep all of these, and please fold them into the existing `%dir %{_cross_libdir}/udev` and `%dir %{_cross_udevrulesdir}` sections.